### DISCUSSION TITLE
Enable trim_doctest_flags in mkdocstrings?

### DISCUSSION THREAD

**Discussion by thatlittleboy** [mkdocstrings/mkdocstrings#384] [Q&A] [answered]
Hi, I'm not sure if this is under the purview of mkdocstrings, but I would like to check if it is possible to get the effect of `trim_doctest_flags` from the [sphinx-doctest ext](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-trim_doctest_flags) using mkdocstrings?

Essentially, with python code like this (excuse the function definition not making sense):
```python
def echo(arg1: str = None):
    """Header docstring.

    Example:

        >>> datetime.date.now()   # doctest: +SKIP
        datetime.date(2008, 1, 1)
        >>> foo('x')
        x

    :param arg1: A string.
    :returns: The original string.
    """
    return arg1
```

Is it possible to achieve such that the `# doctest: SKIP` is removed from the final HTML output? Thanks.

**Comment by pawamoy** [comment, marked as answer]
We could probably add an option to docstrings parsers to remove such comments , yes :slightly_smiling_face:

**Reply by thatlittleboy** [reply]
Got it, thanks. I'll raise this as an issue then.
